docs/src/Link.doc.js

```diff
@@ -17,6 +17,11 @@ card(
 card(
   <PropTable
     props={[
+      {
+        name: 'accessibilitySelected',
+        type: 'boolean',
```

Comment:
@jackhsu978 might be good to follow up and add a `description` - mentioning this prop will update `aria-role` under the hood.